### PR TITLE
Added Locale folder with initial json lang files

### PR DESCRIPTION
**Pull request by DaniellMesquita**

### CODE REVIEW

**Comment by CreaturePhil** [comment]
what is the point of ``en-US``?

File: locales/es.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "buck": ["buck"],
+  "bucks": ["bucks"],
+  "help": ["ayuda"],
+  "name": ["Nombre"],
+  "group": ["Grupo"],
+  "money": ["Dinero"],
+  "lastseen": ["Ultima Visita"],
```
Comment (CreaturePhil) [inline locales/es.json]: why space?

**Comment by DaniellMesquita** [comment]
The point?

**Comment by DaniellMesquita** [comment]
Space? I'm based on your example: http://prntscr.com/97qqy8

**Comment by DaniellMesquita** [comment]
Can U accept because I have other pull to send?

**Comment by DaniellMesquita** [comment]
I've fixed the problem of this PR in the latest commit?

**Comment by DaniellMesquita** [comment]
The space in "Ultima Visita" is not for command, but for the phrase show in user's profile, LOL @CreaturePhil

**Comment by CreaturePhil** [comment]
@DaniellMesquita I don't understand what you mean. The array of strings are suppose to be commands like instead of "Ultima Visita" it is suppose to be "UltimaVisita" for "/ultimavisita""

**Comment by DaniellMesquita** [comment]
@CreaturePhil I'm putting phrases of commands and of chat phrases. Ultima Visita is equal to "Last Seen", message displayed in user's profile!

**Pull request closed without merging**